packages/core/src/schema/models/apiKeys.ts

```diff
@@ -15,12 +15,12 @@ import { workspaces } from './workspaces'
 export const apiKeys = latitudeSchema.table(
   'api_keys',
   {
-    id: bigserial('id', { mode: 'bigint' }).notNull().primaryKey(),
+    id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
     uuid: uuid('uuid')
       .notNull()
       .unique()
       .$defaultFn(() => crypto.randomUUID()),
-    workspaceId: bigint('workspace_id', { mode: 'bigint' })
+    workspaceId: bigint('workspace_id', { mode: 'number' })
```

Comment:
Why?

Reply:
because working with bigints in javascript is a massive pita (they can't be serialized to json) much better to coarce to number it will make our lives easier.

Reply:
I understand but big int are there for some reason. Are we sure we don't want it for possible big tables?

Reply:
i think you misunderstand table column is still big int, we are only coercing to number in javascript land